lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -1721,6 +1721,14 @@ OpFoldResult Torch::ConstantIntOp::fold(FoldAdaptor adaptor) {
   return getValueAttr();
 }
 
+LogicalResult ConstantIntOp::verify() {
+  Type valTy = getValueAttr().getType();
+  if (!valTy.isSignedInteger() && !valTy.isSignlessInteger())
```

Comment:
@silvasean @ramiro050 I've noticed that many places uses `torch.constant.int` with signless integer attributes, eg `i64/i32`, instead of signed integers like `si64/si32`. This isn't causing any issue right now because backends convert that to signless integers, like `arith.constant`. 

I will leave the verifier to check if it's either signed or signless integer for now, but it would be great to be consistent about the attribute type.

Reply:
Let's verify that it is always signless for now.